docs/content/getting-started/_index.md

```diff
@@ -239,3 +239,8 @@ You can optionally customize your broadcast variables individually if you have v
 * `--pfe-broadcasted--link-decoration--focus`
 
 If you include the `pfe-base.css` stylesheet, it will include application of these broadcast variables to links; this is helpful because links are often nested inside of p tags in the content and thus inaccessible to the stylesheet of the web component.
+
+### Avoiding FOUC (Flash of Unstyled Content) with the body unresolved attribute?
+
+PFElements provides a stylesheet that causes the body[unresolved] attribute to avoid FOUC (Flash of Unstyled Content).
```

Comment:
```suggestion
PatternFly Elements provides a stylesheet that causes the `body[unresolved]` attribute to avoid the Flash of Unstyled Content (FOUC).
```